src/views/v2/rsvp/ari/form/title.php

```diff
@@ -11,7 +11,7 @@
  */
 
 ?>
-<h3 class="tribe-tickets__rsvp-ar-form-title tribe-common-h5">
+<header class="tribe-tickets__rsvp-ar-form-title tribe-common-h5">
```

Comment:
Any reason why we aren't using a heading here under the header? If we aren't using a heading, I would suggest at least wrapping the text in a span. The header semantically represents an area that is a header, we should have a wrapper where we can target the actual text (and move the title styles to the wrapper).

Also apply to the template.